resources/views/formfields/relationship.blade.php

```diff
@@ -160,7 +160,7 @@ class="form-control @if(isset($options->taggable) && $options->taggable == 'on')
 
                         @php
                             $selected_values = isset($dataTypeContent) ? $dataTypeContent->belongsToMany($options->model, $options->pivot_table)->get()->map(function ($item, $key) use ($options) {
-                                return $item->{$options->label};
+                                return $item->{$options->key};
```

Comment:
Fixes #3457.

Reply:
As a side note: those keywords (fixes, resolves, ...) only have an effect when used in the first comment.

Reply:
Thank you. n00b is as n00b does.